### ISSUE TITLE
Access last four of credit card

### ISSUE THREAD

**Issue by markamccorkle** [braintree/braintree-ios-drop-in#216]
### General information

* SDK/Library version: 
8.0.0
* Environment: 
Sandbox
* iOS Version and Device: 
13.3 iPhone 11 Pro Max, All simulators
* Integration type and version: 
Carthage 0.34.0

### Issue description
I am trying to access the `lastFour` variable found nested deeply in the BTDropInResult but the objects included in this SDK don't allows access to those variables (the look private). What am I missing to access those variable? I can see them in the debugger if I drill down in the tree, but looking at the actual objects in the SDK, those variables do not exist. I have tried assigning each nested object to their associated Object types as well. 

What's strange is when I view the result.paymentMethod the object is actually a `BTCardNonce`, not a `BTPaymentMethodNonce` like is expected.

For instance, **threeDSecureJSON** does not exist on the `threeDSecureInfo` object if you go look in the `BTThreeDSecureInfo.h` found in `BraintreeCard.framework`

Code I am trying
```
        let dropIn = BTDropInController(authorization: key, request: request)
        { (controller, result, error) in
            if error != nil {
                self.delegate?.dropInError(error)
            } else if result?.isCancelled == true {
                self.delegate?.dropInCancelled()
			} else if let result: BTDropInResult = result {
                if result.paymentOptionType == .applePay {
                    // Apple pay specific callback
                    // Go out and get the nonce from apple to use for returning to the server
                    self.requestApplePayPayment(clientToken: key, items: summaryItems)
                } else {
			if let paymentM: BTCardNonce = result.paymentMethod as? BTCardNonce {
				let threeDS = paymentM.threeDSecureInfo // This object has no threeDSSecureJSON attached to it
```

```
result	BTDropInResult *	0x600001339000	0x0000600001339000
NSObject	NSObject	
_cancelled	BOOL	NO	false
_paymentOptionType	BTUIKPaymentOptionType	BTUIKPaymentOptionTypeVisa
_paymentMethod	BTCardNonce *	0x600003640a20	0x0000600003640a20
BTPaymentMethodNonce	BTPaymentMethodNonce	
_cardNetwork	BTCardNetwork	BTCardNetworkVisa
_lastTwo	NSTaggedPointerString *	@"11"	0xef26504b78098b1c
_bin	NSTaggedPointerString *	@"411111"	0xef2543586b198b48
_binData	BTBinData *	0x60000328eb70	0x000060000328eb70
_threeDSecureInfo	BTThreeDSecureInfo *	0x600001331960	0x0000600001331960
NSObject	NSObject	
_errorMessage	NSString *	nil	0x0000000000000000
_threeDSecureJSON	BTJSON *	0x600001331940	0x0000600001331940
NSObject	NSObject	
_value	__NSDictionaryI *	11 key/value pairs	0x0000600002b5a140
[0]	(null)	@"securityQuestions" : @"1 element"	
[1]	(null)	@"details" : 6 key/value pairs	
key	NSTaggedPointerString *	@"details"	0xe81096dd6f4cce49
value	__NSDictionaryI *	6 key/value pairs	0x0000600003c11100
[0]	(null)	@"cardType" : @"Visa"	
[1]	(null)	@"expirationYear" : @"2022"	
[2]	(null)	@"lastTwo" : @"11"	
[3]	(null)	@"lastFour" : @"1111"	
[4]	(null)	@"bin" : @"411111"	
[5]	(null)	@"expirationMonth" : @"02"	
```

**Comment by billwerges** [comment]
Hi @markamccorkle,

It looks like we're missing that value as a public facing property on `BTCardNonce`. We'll look into getting that added and update when it's available.

**Comment by markamccorkle** [comment]
Excellent. Thank you!

Thank you,

Mark McCorkle
President - edibu
(321) 251-1119<tel:(321)%20251-1119>
www.edibu.com<http://www.edibu.com/>

On Mar 5, 2020, at 3:51 PM, billwerges <notifications@github.com> wrote:

﻿

Hi @markamccorkle<https://github.com/markamccorkle>,

It looks like we're missing that value as a public facing property on BTCardNonce. We'll look into getting that added and update when it's available.

**Comment by markamccorkle** [comment]
@billwerges Any progress on getting the lastFour accessible?

**Comment by scannillo** [comment]
Hi @markamccorkle. We recently made this fix and are looking to release it this week. We will update you when!

**Comment by scannillo** [comment]
Hi @markamccorkle - this has been released in v4.33.0 of Braintree iOS (https://github.com/braintree/braintree_ios/releases/tag/4.33.0)

**Comment by markamccorkle** [comment]
> Hi @markamccorkle - this has been released in v4.33.0 of Braintree iOS (https://github.com/braintree/braintree_ios/releases/tag/4.33.0)

Awesome! Thank you!!!
